boards/cmod_a7/board_specific.tcl

```diff
@@ -1,3 +1,3 @@
 set fpga_board  cmod_a7
-set part_name   xc7a15tcpg236-1 # xc7a100tcsg324-1
-set hw_device   xc7a15t_0       # xc7a100t_0
+set part_name   xc7a15tcpg236-1
```

Comment:
The comments have been removed because Vivado throws up errors when they are there